au4/src/project/view/projectpropertiesmodel.cpp

```diff
@@ -31,14 +31,29 @@ using namespace au::project;
 ProjectPropertiesModel::ProjectPropertiesModel(QObject* parent)
     : QAbstractListModel(parent)
 {
-    m_project = context()->currentProject();
+}
+
+void ProjectPropertiesModel::init()
+{
+    m_project = globalContext()->currentProject();
+
+    globalContext()->currentProjectChanged().onNotify(this, [this]() {
+        m_project = globalContext()->currentProject();
+    });
+
+    if (!m_project) {
+        return;
+    }
+
     // if (project) {
     //     m_projectMetaInfo = project->metaInfo();
     // }
```

Comment:
I lost the point, where do we get the properties now?
And when changing the project in the global context, we need to get new properties and call the load.

Reply:
Right now we're not using the properties yet, see the commented metaInfo() function.
I've used the properties model to retrieve path for temporary thumbnail image in QML, we'll need to update this model later when we're going to have some project preferences UI to change meta info (and the thumbnail image path will change too as we implement this one: https://github.com/audacity/audacity/issues/6996 )